### PR TITLE
create static categories drink page

### LINKED ISSUE #25: [HTML- CSS-Responsive] Categories-Drink Page
- Design
Categories-drinks

### PR DESCRIPTION
**Pull request by MohammedDahalan**
Relates #25

### CODE REVIEW

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
edit style for tablet and desktop screen

**Review by muhannaAtalla20 — APPROVED** [review]

File: otherPages/categories/categoriesDrink.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      <a href=""><div class="carousel-cell">Shot</div></a>
+      <a href=""><div class="carousel-cell">Coffee / Tea</div></a>
+      <a href=""><div class="carousel-cell">Homemade Liqueur</div></a>
+      <a href=""><div class="carousel-cell">Punch</div></a>
+      <a href=""><div class="carousel-cell">Beer</div></a>
+      <a href=""><div class="carousel-cell">Soda</div></a>
+    </section>
+    <main class="categories">
+      <a href="">
+        <div class="category" style="background-image: url(../../assets/images/cateories/drink/Ordinary\ Drink.jpg)">
```
Comment (yaqootturman) [inline otherPages/categories/categoriesDrink.html]: you shouldn't add style here

File: otherPages/categories/categoriesDrink.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <a href="../home/home.html"> <i class="fas fa-arrow-alt-circle-left"></i></a>
+      <img class="logo" src="../../assets/images/logo.png" alt="" />
+    </header>
+    <section class="title">
+      <span>Categories</span>
+    </section>
+    <section class="carousel" data-flickity='{ "groupCells": true }'>
```
Comment (yaqootturman) [inline otherPages/categories/categoriesDrink.html]: No need for slides if its not work correctly

File: otherPages/categories/categoriesDrink.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    />
+    <link rel="stylesheet" href="carouselLibrary\carousel.css" media="screen" />
+    <link rel="shortcut icon" href="../../assets/images/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="categories.css" />
+    <title>TOP CHEF</title>
+  </head>
+  <body>
+    <header class="header">
+      <a href="../home/home.html"> <i class="fas fa-arrow-alt-circle-left"></i></a>
+      <img class="logo" src="../../assets/images/logo.png" alt="" />
```
Comment (yaqootturman) [inline otherPages/categories/categoriesDrink.html]: add alt

File: otherPages/categories/categoriesDrink.html [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="carouselLibrary\carousel.css" media="screen" />
+    <link rel="shortcut icon" href="../../assets/images/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="categories.css" />
+    <title>TOP CHEF</title>
```
Comment (yaqootturman) [inline otherPages/categories/categoriesDrink.html]: this title is for this page not for the app

**Review by yaqootturman — CHANGES_REQUESTED** [review]
change the files name for camelCase convention

File: otherPages/categories/categoriesDrink.html (hunk 1 above)
Comment (IsraaHamdi) [reply otherPages/categories/categoriesDrink.html]: using inline style there because this image fetch from api  when create the element using dom

**Review by IsraaHamdi — APPROVED** [review]

**Review by farahalzard — APPROVED** [review]

**Review by yaqootturman — CHANGES_REQUESTED** [review]
change files name

**Comment by IsraaHamdi** [comment]
> change files name

what the files needed to change name

**Comment by IsraaHamdi** [comment]
> > change files name
> 
> what the files needed to change

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**